### ISSUE TITLE
Does Press This belong in tools?

### ISSUE THREAD

**Issue by MichaelArestad** [MichaelArestad/Press-This#16]
Or would it be better elsewhere?

Maybe the main dashboard?

**Comment by stephdau** [comment]
Maybe somewhere in the new/edit post screens as well.

**Comment by marcelomazza** [comment]
I agree that Press This should be elsewhere, and the Dashboard makes sense.

About putting it on the new/edit post **a quick idea**: if the user is publishing a certain type of Post Format (i.e: Link), why can add a dynamic callout as a remember that Press This exists and if you are posting a link it would be probably easier to do it with it.

**Comment by stephdau** [comment]
Related: https://wordpress.slack.com/archives/feature-pressthis/p1422957632000884
See also: https://core.trac.wordpress.org/ticket/28873

**Comment by stephdau** [comment]
Posterity note that we've cerated our own screen (though still under tools) under `/wp-admin/tools.php?page=press_this_options`, in the meantime, to address accessibility, etc.
